src/back/hlsl/writer.rs

```diff
@@ -100,6 +100,9 @@ impl<'a, W: Write> Writer<'a, W> {
             }
         }
 
+        // Extra newline for readability
+        writeln!(self.out)?;
```

Comment:
@kvark these seem to be placed at the beginning of the file, was this supposed to go somewhere else?

Reply:
These are placed after the constants. Previously, each constant had its own newline.

Reply:
The diff shows it placed at the beginning of the files though, unless the diff is unrelated

Reply:
Oh, that's probably because those tests don't have constants. That's ok. We shouldn't optimize the verbosity for small shaders, but rather focus on bigger ones.